readme.md

```diff
@@ -1,151 +1,131 @@
 ![Mend Renovate CLI banner](https://docs.renovatebot.com/assets/images/mend-renovate-cli-banner.jpg)
 
-# Renovate
-
-Automated dependency updates.
-Multi-platform and multi-language.
-
 [![License: AGPL-3.0-only](https://img.shields.io/badge/license-%20%09AGPL--3.0--only-blue.svg)](https://raw.githubusercontent.com/renovatebot/renovate/main/license)
 [![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![Build status](https://github.com/renovatebot/renovate/actions/workflows/build.yml/badge.svg)](https://github.com/renovatebot/renovate/actions/workflows/build.yml)
 ![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/renovatebot/renovate/badge)](https://securityscorecards.dev/viewer/?uri=github.com/renovatebot/renovate)
 
-## Why Use Renovate?
+# What is the Mend Renovate CLI?
+
+Renovate is an automated dependency update tool.
+It helps to update dependencies in your code without needing to do it manually.
+When Renovate runs on your repo, it looks for references to dependencies (both public and private) and, if there are newer versions available, Renovate can create pull requests to update your versions automatically.
+
+## Features
+
+- Delivers update PRs directly to your repo
+  - Relevant package files are discovered automatically
```

Comment:
```suggestion
  - Renovate automatically finds nearly all relevant package files
```

Renovate mostly finds all the relevant package files automatically. But sometimes users must tell Renovate where/how to find some package files.

I'll let you decide if the promise is: "Renovate finds all package files" or "Renovate finds most package files".

Reply:
Thanks for the suggestion. I don't think we need to drill into details about edge cases (ie. some packages need help) in the overview page. And whether it's "all" or "most", the short statement "package files are discovered" covers the point and is still true.